### ISSUE TITLE
About some problem

### ISSUE THREAD

**Issue by dlyldxwl** [farmingyard/caffe-mobilenet_v2#2]
Thanks for you job!
After I looked at the prototxt you wrote, the comparison paper found several differences.
First,in res7_conv1 layer, the input is 28×28×32, I think channel of the layer is 32×6，but your prototxt is 144.And the same problem occurs in the res14_conv1 layer. Can you tell me the reason? Second, I believe you wrote the prototxt according to the input column in Table 2 of the paper, ignored s column, however, the paper wrote that mobilenet v2  has 19 bottlenect layers rather 17. how do you think?
Last, can you tell me the result of your training?
Thanks again !

**Comment by farmingyard** [comment]
@dlyldxwl 
You are right, I have fix the channel num problems. For the second problem, maybe the paper takes the first and last conv as bottleneck layers.

**Comment by dlyldxwl** [comment]
@farmingyard
Thank you for you reply~
Maybe for this reason, paper claim mobilenet v2 has 19 bottleneck layers~
I expect you to train a good performance of the mobilenet V2 classification network, then I can finetune it to get mobilenet V2_ssd network.

**Comment by dlyldxwl** [comment]
@farmingyard 
 May be the reason that leads to bad result is relu6 rather than relu？
